Backend/updateBackendEnv.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 pip install virtualenv
-virtualenv env
+virtualenv -p python2.6 env
```

Comment:
That's the root of this whole problem. The mod_wsgi on the apache server that's launching this was compiled with 2.6 which is why our app has always been running using the python2.6 interpreter (Remember that problem with the set syntax?). The server currently has a smattering of tools for python2.6 and python2.7 and it is hard to know which ones are being used when and where. By explicitly forcing the virtualenv on the server to create a python2.6 virtualenv the right tools are being used for everything (by tools I mean things like pip) and they mesh correctly with the mod_wsgi. 

The goal is that once we get the new server up and running (Configured correctly with python2.7) we will be able to switch over and everything will be running 2.7.